config/locales/ja.yml

```diff
@@ -284,8 +310,11 @@ ja:
     show_more: もっと見る
     visibilities:
       private: 非公開 - フォロワーだけに公開
```

Comment:
Thank you for your review.
I think too. I will fix to remove  description after dash from `public`, `unlisted`, and `private`.